cmd/kogito/command/install/kogito_operator.go

```diff
@@ -64,6 +66,10 @@ func (i *installKogitoOperatorCommand) RegisterHook() {
 		PreRun:  i.CommonPreRun,
 		PostRun: i.CommonPostRun,
 		Args: func(cmd *cobra.Command, args []string) error {
+			ch := i.flags.channel
+			if !shared.IsChannelValid(ch) {
+				return fmt.Errorf("Invalid Kogito channel type : %s ", ch)
```

Comment:
maybe add the valid channels in the error as well?

Reply:
done